src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started working");
+    }
+
+    public void stopWork() {
+        System.out.println("Excavator stopped working");
```

Comment:
The message in the 'stopWork' method should be more informative. It should indicate that it's an 'Excavator' that stopped working, which you've done correctly. However, consider including additional details if applicable, such as the reason for stopping or the task it was performing.